4.sc_Morphology_Neural_Network_MLP_Model/0.model_training/scripts/Hyperparameter_Optimization_model_regression.py

```diff
@@ -401,15 +453,21 @@
 )
 df_values_Y = df_values["oneb_Metadata_Treatment_Dose_Inhibitor_Dose"]
 
-# %%
+
+# In[31]:
+
+
 test_data = Dataset_formatter(
     torch.FloatTensor(df_values_X.values), torch.FloatTensor(df_values_Y.values)
 )
 
 # convert data class into a dataloader to be compatible with pytorch
 test_loader = torch.utils.data.DataLoader(dataset=test_data, batch_size=1)
 
-# %%
+
+# In[32]:
+
+
 # calling the testing function and outputting list values of tested model
 if params.MODEL_TYPE == "Multi_Class" or params.MODEL_TYPE == "Regression":
```

Comment:
```suggestion
if any(params.MODEL_TYPE == model_type for model_type in ["Multi_Class", "Regression"]):
```
reduce code duplication